elliott/elliottlib/cli/find_bugs_non_first_fix_cli.py

```diff
@@ -0,0 +1,113 @@
+import sys
+import traceback
+from typing import Dict, Iterable, List
+
+import click
+from artcommonlib import logutil
+
+from elliottlib import Runtime
+from elliottlib.bzutil import Bug, BugTracker, is_first_fix_any
+from elliottlib.cli.common import cli
+from elliottlib.cli.find_bugs_sweep_cli import FindBugsMode
+
+LOGGER = logutil.get_logger(__name__)
+
+
+class FindBugsNonFirstFix(FindBugsMode):
+    def __init__(self):
+        super().__init__(
+            status={'MODIFIED', 'VERIFIED', 'ON_QA'},
+            cve_only=True,
+        )
+
+
+@cli.command(
+    "find-bugs:non-first-fix", short_help="Closes trackers if they are NOT first fix when branch is pre-release"
+)
+@click.option("--noop", "--dry-run", is_flag=True, default=False, help="Don't change anything")
+@click.pass_obj
+def find_bugs_non_first_fix_cli(runtime: Runtime, noop):
+    """Checks if CVE trackers in a pre-release branch [before GA] are not 'first fix',
+    and if so, they will be closed
+    \b
+
+        $ elliott -g openshift-4.Y find-bugs:non-first-fix
+    """
+    runtime.initialize()
+    find_bugs_obj = FindBugsNonFirstFix()
+    exit_code = 0
+    for b in [runtime.get_bug_tracker('jira')]:
+        try:
+            find_bugs_non_first_fix(runtime, find_bugs_obj, noop, b)
+        except Exception as e:
```

Comment:
the try-except block was added when there were multiple bug_trackers to run for. when there is a single one, we don't really need this try-except pattern, including setting the exit_code explicitly. so we can remove them